client/state/push-notifications/actions.js

```diff
@@ -292,8 +295,10 @@ export function toggleEnabled() {
 		} );
 		if ( enabling ) {
 			dispatch( fetchAndLoadServiceWorker() );
+			tracks.recordEvent( 'calypso_web_push_notification_enabled' );
```

Comment:
@jblz is this the right place for this? I wasn't sure the best place to put this, since there's no longer a callback